okhttp/src/jvmMain/kotlin/okhttp3/internal/http/CallServerInterceptor.kt

```diff
@@ -151,10 +151,10 @@ class CallServerInterceptor(private val forWebSocket: Boolean) : Interceptor {
   private fun shouldIgnoreAndWaitForRealResponse(code: Int, exchange: Exchange): Boolean = when {
     // Server sent a 100-continue even though we did not request one. Try again to read the
     // actual response status.
-    code == 100 -> true
+    code == HTTP_CONTINUE -> true
 
-    // Early Hints (103) but not supported yet in OkHttp
-    code == 103 -> true
+    // Processing & Early Hints (103) but not supported yet in OkHttp
```

Comment:
The comment is slightly misleading IMHO. Supporting 1xx codes means doing the same thing for all of them, except, well, for the exceptions (being 100 and 101).

It just means consuming the response, until final response is received (and potentially making the information about it available to the caller).